### DISCUSSION TITLE
Problem with connecting in SqlServer

### DISCUSSION THREAD

**Discussion by leonardolauren** [airbytehq/airbyte#32730] [Connector Questions]
Could not connect with provided configuration. Error: Cannot invoke "com.fasterxml.jackson.databind.JsonNode.asText()" because the return value of "com.fasterxml.jackson.databind.JsonNode.get(String)" is null

2023-11-22 13:46:02 [32mINFO[m i.a.w.t.TemporalAttemptExecution(get):126 - Cloud storage job log path: /workspace/fb8d00d9-231f-46cc-a594-2ff1bd14ddff/0/logs.log

I was trying the connection but  says these messages up.

**Comment by marcosmarxm** [comment]
Can you share a config example and what parameters are you using?
Are you setting up a source or destination?

[a comment by an automated account is not shown]
